app/models/submission.rb

```diff
@@ -1,6 +1,7 @@
 class Submission < ActiveRecord::Base
   belongs_to :student
   belongs_to :repo
+  belongs_to :user
```

Comment:
We could make this somewhat more informative by calling the attribute `feedback_provider`:
```ruby
belongs_to :feedback_provider, class_name: 'User'
```